packages/kbn-interpreter-utils/ui/lib/socket.js

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
i need to remove this file, seems something wen't wrong with the move, its now in core_plugins/interpreter/public/lib/socket.js